### DISCUSSION TITLE
Receiver can't receive any information?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#795] [Q&A]
The example I use is [here](https://github.com/buzhiqiuyuebai/A319/tree/main),(Since the upload of the github compressed package failed, I uploaded the file `Merge "A319stl0.04m.msh"` in loh.geo to  [here](https://gitlab.onelab.info/gmsh/gmsh/-/issues/2197#note_19592:~:text=A319stl0.04m.msh%20is%20here.A319stl0.04m.zip))

But in output file, the receiver seems can't receiver any information as below picture.
![image](https://user-images.githubusercontent.com/114474894/218404168-96537779-44ef-4d54-b21b-6b24428ddae4.png)

The LOH3 that was running before can receive information. If the information is received in the receiver, does the code support deleting the point source and use the information received by the receiver as the excitation source for reverse time migration simulation? Also, for the same grid file, the code works fine when I set cl_1 = 5; but when I set cl_1 = 1(`SeisSol_Release_shsw_6_elastic`), the code encounters an error( ([order6Error](https://github.com/buzhiqiuyuebai/A319/blob/main/A319order6Error)) at runtime, also I set CFL=0.35 as you [suggested](https://github.com/SeisSol/SeisSol/discussions/737#:~:text=Maintainer-,try%20CFL%20%3D%200.35%2C%20to%20my%20experience%20CFL%3D0.5%20is%20too%20large%20for%200.6%20(probably%200.45%20is%20enough%20but%200.35%20puts%20you%20on%20the%20safe%20side.,-Unmark%20as%20answer) before, but it still doesn't work, running this example with 3rd order basis functions doesn't have this problem. When using the sixth-order basis function calculation, there will be problems for some examples, but there may be no problems for other examples. What is the reason for this? And when I set cl_1 = 0.6,  There is a problem（[pumgenError](https://github.com/buzhiqiuyuebai/A319/blob/main/pumgenError)） with format conversion in pumgen, do you know what caused this?

**Comment by Thomas-Ulrich** [comment]
Probably because the following option is not set
```
! (Optional) Synchronization point for receivers.
!            If omitted, receivers are written at the end of the simulation.
ReceiverOutputInterval = 10.0
```
?
